extensions/lifecycle/tasks/LifecycleDeleteObjectTask.js

```diff
@@ -130,11 +137,18 @@ class LifecycleDeleteObjectTask extends BackbeatTask {
                 return done(errors.InternalError
                     .customizeDescription('Unable to obtain client'));
             }
+
+            LifecycleMetrics.onLifecycleStarted(log, 'expiration:mpu',
+                location, startTime - transitionTime);
+
             req = s3Client[reqMethod](reqParams);
         }
         attachReqUids(req, log);
         return req.send(err => {
             LifecycleMetrics.onS3Request(log, reqMethod, 'expiration', err);
+            LifecycleMetrics.onLifecycleCompleted(log,
+                actionType === 'deleteMPU' ? 'expiration:mpu' : 'expiration',
```

Comment:
Is there a reason not to follow the same case rule ? shouldn't it be `delete:mpu` or `expirationMPU` ?

Reply:
this follows the same kind of convention as bucket notifications, and thus makes MPU expiration a kind of "sub-class" of expirations. Specifically, this allows having metrics for all expirations, by filtering on `type =~ '^expiration'`.